source/installation.rst

```diff
@@ -0,0 +1,128 @@
+.. _install:
+
+
+=============================================     
+Installing |pdmdb|
+=============================================
+
+.. contents::
+   :local:
+
+Percona provides installation packages for the |pdmdb|
+in |deb| and |rpm| formats for the most 64-bit Linux distributions.
+
+.. list-table:: 
+   :widths: 100 
+   :header-rows: 1
+
+   * - Supported Distributions
+   * - * Debian 9 (stretch)
+       * Debian 10 (buster)
+       * Ubuntu 16.04 (Xenial Xerus)
+       * Ubuntu 18.04 LTS (Bionic Beaver)
+       * Ubuntu 20.04 (Focal Fossa)
+   * - * Red Hat Enterprise Linux / CentOS 6
+       * Red Hat Enterprise Linux / CentOS 7
+       * Red Hat Enterprise Linux / CentOS 8
+
+|pdmdb| is installed using the |percona-release| utility.
+This is the most straightforward way since |percona-release|
+enables the required repositories for you.
+
+The installation of |pdmdb| includes the following steps:
+
+#. Install the |percona-release| utility.
+#. Set up the |Percona| repository for the required version of
+   |pdmdb|.
+#. Install the packages.
+
+.. note::
+
+   Make sure to run all commands as root or via |sudo| during the installation.
+
+Install |percona-release|
+==============================================
+
+See the `Percona Software Repositories Documentation <https://www.percona.com/doc/percona-repo-config/percona-release.html#installation>`_
+for installation instructions relevant to your operating system.
+
+If you have previously installed |percona-release|, make sure to update it to the
+latest version. Refer to the `documentation <https://www.percona.com/doc/percona-repo-config/percona-release.html#updating-percona-release-to-the-latest-version>`_
+for details.
+
+Set up |Percona| repository
+==============================================
+
+As soon as |percona-release| is up-to-date, set up |Percona| repository for the
+required version of |pdmdb| (``pdmdb-<version>``). For example, to set up |pdmdb|
+4.2.6, use the following command as root or via |sudo|:
+
+.. code-block:: bash
+
+   $ percona-release setup pdmdb-4.2.6
```

Comment:
do we need to mention that root permissions are required? @borys-belinsky what do you think